File: OOPS_LAB/Program_5/account.js
class Account{
    accountNumberGen()
    {
        var num = Math.random()*10000000000
        num = Math.round(num)
        return num
    }

    balanceGen()
    {
        var num = Math.random()*10000
        num = Math.round(num)
        return num
    }
    constructor(name = "")
    {
        this.HolderName = name
        this.AccountNumber = this.accountNumberGen()
        this.balance = this.balanceGen()
    }

    display()
    {
        console.log(`Name : ${this.HolderName}\nAccount No : ${this.AccountNumber}\nBalance : ${this.balance}`)
    }
}


const user1 = new Account("Ashfak")
user1.display()

File: OOPS_LAB/Program_5/account.cpp
#include <iostream>
#include <string>
#include <cstdlib>
#include <ctime>
#include <cmath>

using namespace std;

class AccountGen {
private:
    string HolderName;
    int AccountNumber;
    double Balance;

    int AccountNumGen() {
        return rand() % 100000000;
    }

    double BalanceGen() {
        double bal = (rand() % 1000000) / 100.0;
        return round(bal);
    }

public:
    AccountGen(string name) {
        HolderName = name;
        AccountNumber = AccountNumGen();
        Balance = BalanceGen();
    }

    void display() {
        cout << "Name : " << HolderName <<endl;
        cout << "Account Number : " << AccountNumber <<endl;
        cout << "Balance : " << Balance <<endl;
    }
};

int main() {
    srand(time(0)); 
    
    AccountGen user1("Ashfak");
    user1.display();

    return 0;
}